creature/D&D Wiki; Rabbit and Squirrel.json

```diff
@@ -0,0 +1,160 @@
+{
+    "_meta": {
+        "sources": [
+            {
+                "json": "dndwiki_rabbit",
+                "abbreviation": "D&D Wiki",
+                "full": "D&D Wiki - Rabbit & Squirrel",
+                "url": "https://www.dandwiki.com/wiki/Rabbit_(5e_Creature)",
+                "authors": [
+                    "Various Contributors"
+                ],
+                "convertedBy": [
+                    "ldsmadman#8990"
+                ]
+            }
+        ],
+        "dateAdded": 1590443220,
+        "dateLastModified": 1631151467
+    },
+    "monster": [
+        {
+            "name": "Rabbit",
+            "source": "dndwiki_rabbit",
+            "size": "T",
+            "type": "beast",
+            "alignment": [
+                "U"
+            ],
+            "ac": [
+                11
+            ],
+            "hp": {
+                "average": 1,
+                "formula": "1d4 - 2"
+            },
+            "speed": {
+                "walk": 35,
+                "burrow": 5
+            },
+            "str": 1,
+            "dex": 13,
+            "con": 7,
+            "int": 2,
+            "wis": 10,
+            "cha": 3,
+            "skill": {
+                "perception": "+2"
+            },
+            "passive": 12,
+            "action": [
+                {
+                    "name": "Bite",
+                    "entries": [
+                        "{@atk mw} {@hit 3} to hit, reach 5 ft., one target. {@h}1 piercing damage."
+                    ]
+                }
+            ],
+            "damageTags": [
+                "P"
+            ],
+            "miscTags": [
+                "MW"
+            ],
+            "uniqueId": "7cbe3f0531e940888e7903fbb6032ceb",
```

Comment:
Be aware that `uniqueId` gets created automatically when the brew is loaded, and is not needed in the homebrew file.

Unsure if it would cause any issues.

Reply:
Yeah, those shouldn't be in there. They'll get clobbered as it gets loaded, either way.